kubernetes/helm_charts/logging/kibana/templates/deployment.yaml

```diff
@@ -79,15 +79,6 @@ spec:
         image: "{{ .Values.image }}:{{ .Values.imageTag }}"
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         env:
-          {{- if .Values.elasticsearchURL }}
-          - name: ELASTICSEARCH_URL
-            value: "{{ .Values.elasticsearchURL }}"
-          {{- else if .Values.elasticsearchHosts }}
-          - name: ELASTICSEARCH_HOSTS
-            value: "{{ .Values.elasticsearchHosts }}"
-          {{- end }}
-          - name: SERVER_HOST
```

Comment:
add a if-else block and raise a PR. Put the pr here as comment. In future when we upgrade the helm chart, we should be overwrite it with upstream without any manual change

Reply:
@rjshrjndrn This changes are specifically for kibana to point to multiple es nodes. If it has hosts in env it will read it here instead of that we are passing it in kibana config file.

Reply:
@harshavardhanc for that we can keep the hosts as empty. and in values.yaml we can override SERVER_HOST as 0.0.0.0

Reply:
@rjshrjndrn this is done